docs/generate_docs.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# script for generating documentation for NMODL
+
+set -eu
+
+if [ $# -lt 2 ]
+then
+    echo "Usage: $(basename "$0") output_dir python_exe [use_virtual_env]"
+    exit 1
+fi
+
+# the dir where we put the temporary build and the docs
+output_dir="$1"
+# path to the Python executable
+python_exe="$2"
+use_venv="${3:-}"
+
+if ! [ -d "${output_dir}" ]
+then
+    mkdir -p "${output_dir}"
+fi
+
+build_dir="build"
+docs_dir="docs"
+
+echo "== Building documentation files in: ${output_dir}/${docs_dir} =="
+echo "== Temporary project build directory is: ${output_dir}/${build_dir} =="
+
+if [ "${use_venv}" != "false" ]
+then
+    venv_name="$(mktemp -d)"
```

Comment:
Does implies you need to reinstall the `venv` every time even if you're reusing the directory.